CMakeLists.txt

```diff
@@ -25,6 +25,8 @@ option(FILAMENT_SUPPORTS_XCB "Include XCB support in Linux builds" ON)
 
 option(FILAMENT_SUPPORTS_XLIB "Include XLIB support in Linux builds" ON)
 
+option(CATALYST "Build for Mac Catalyst in iOS builds" OFF)
```

Comment:
Not needed after the changes above.

Reply:
74fd2bd3fffbecb601faa377ae5c761f5d0b8530